akka-http-core/src/main/scala/akka/http/impl/util/JavaMapping.scala

```diff
@@ -123,7 +123,7 @@ private[http] object JavaMapping {
   def javaToScalaAdapterFlow[J, S](implicit mapping: JavaMapping[J, S]): scaladsl.Flow[J, S, Unit] =
     scaladsl.Flow[J].map(mapping.toScala(_))
   def adapterBidiFlow[JIn, SIn, SOut, JOut](implicit inMapping: JavaMapping[JIn, SIn], outMapping: JavaMapping[JOut, SOut]): scaladsl.BidiFlow[JIn, SIn, SOut, JOut, Unit] =
-    scaladsl.BidiFlow.wrap(javaToScalaAdapterFlow(inMapping), scalaToJavaAdapterFlow(outMapping))(scaladsl.Keep.none)
+    scaladsl.BidiFlow.fromGraphsMat(javaToScalaAdapterFlow(inMapping), scalaToJavaAdapterFlow(outMapping))(scaladsl.Keep.none)
```

Comment:
Does not look very nice but at least it's *very* visually recognisable - "oh, fromX[Mat], I know that."